application/__init__.py

```
@@ -18,14 +18,25 @@
 import sys
 
+import datetime
+
+from clp_logging.handlers import CLPFileHandler
+from pathlib import Path
 import yaml
 from flask import Flask, Blueprint, jsonify
 from werkzeug.exceptions import HTTPException
 from werkzeug.serving import WSGIRequestHandler
 
+timestamp = datetime.datetime.now()
```

Ruff (0.8.2)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)